FCCee/CLD/compact/CLD_o2_v05/CLD_o2_v05.xml

```diff
@@ -56,27 +56,34 @@
 
         <constant name="DetID_HOMAbsorber"          value=" 19"/>
 
-        <constant name="BeamPipeWidth"   	value="1.2*mm" />
-	<constant name="BeamPipeWidthFirstCone"   	value="1.0*mm" />
-        <constant name="CentralBeamPipe_zmax" value="12.5*cm"/>
-        <constant name="BePartEnd_z" value="90.0*cm"/>
-	<constant name="SeparatedBeamPipe_z" value="1159.97*mm"/>
+        <constant name="BeamPipeWidth"   	value="1.7*mm" />
```

Comment:
```suggestion
        <constant name="BeamPipeWidth" value="1.7*mm" />
```

Reply:
This way aligns with the next line. Won't fix.